source/agora/consensus/data/Transaction.d

```diff
@@ -53,6 +55,9 @@ public struct Transaction
     /// The list of newly created outputs to put in the UTXO
     public Output[] outputs;
 
+    /// The data to store
+    public TransactionData data;
+
```

Comment:
This adds > 1kb of data to *every single transaction*. Not something acceptable.